mlrun/api/schemas/feature_store.py

```diff
@@ -39,6 +39,7 @@ class Config:
 
 
 class FeatureSet(BaseModel):
+    kind: str = "FeatureSet"
```

Comment:
1. make it an enum
2. Move it to either metadata or spec

Reply:
Also this looks like a field that we'll want to filter by in the future, so add it a column in the DB

Reply:
1. Will do
2. In terms of the location of the field in the object document, this field is located in the top level - look at k8s objects for example. Therefore this field needs to be here and not in md/spec. As a matter of fact, if `kind` was in metadata then this fix would not be needed since the `Extra.allow` would maintain it in the DB
3. As for filtering - I'm not sure, as this field is pretty much required to have a single value per resource type. Therefore, the only way we will query on it is if we add a DB-wide query facility that will query across all objects. Either way, adding this as a field serves no purpose

Reply:
talked offline, I was thinking this is to represent some optional subkind for feature-set - no, this is the kind of the object, which obviously can only be one for the `FeatureSet` schema - `FeatureSet`